src/utils/general.js

```diff
@@ -8,11 +8,17 @@ let count = 0
  */
 const fixIndentation = str => {
   // Get whitespaces
-  let match = str.match(/^[ \t]*(?=\S|$)/gm)
-  // Remove first, empty item
+  const match = str.match(/^[ \t]*(?=\S|$)/gm)
+
+  // Handle oneline TTL case
+  if (match.length === 1) {
+    return {
+      text: str,
+      indentColumns: 0
+    }
+  }
+  // Remove first, empty item, and if it is a oneline TTL we already handled it
   match.splice(0, 1)
-  // Remove empty lines
-  match = match.map(x => (x.length < match[match.length - 1].length ? '  ' : x))
 
   if (!match) {
```

Comment:
This should probably be `match.length === 0`? Even empty arrays are truthy so this condition is never hit.